src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const object of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

And also destructure `action`